lib/History.js

```diff
@@ -10,13 +10,25 @@ class History extends Request {
 
   // GET /api/history/period/<timestamp>
   // https://home-assistant.io/developers/rest_api/#get-apihistoryperiodlttimestamp
-  state(timestamp, filter) {
-    if (moment(timestamp, 'YYYY-MM-DDTHH:mm:ssZ', true).isValid() === false) {
+  state(timestamp, filter, endTimestamp) {
```

Comment:
This should be a breaking change to the module. If we want to make the timestamps optional, they should be the second and third parameters, with the `filter` being the first.

```
state(filter, startTimestamp, endTimestamp) { ... }
```

Reply:
I'm not sure whether we this should really be a breaking change just because it looks nicer to us? In the end all parameters are optional, so it's no must have param which should be written first.

Reply:
I'd like to see the order changed for sanity sake. We'll bump the major version after the change has been made.